jax/lax/lax.py

```diff
@@ -1601,6 +1601,29 @@ def naryop(result_dtype, accepted_dtypes, name, translation_rule=None):
 standard_naryop = partial(naryop, _input_dtype)
 
 
+def broadcast_translate(translate):
```

Comment:
I'd add a brief function-level comment explaining what this function as a whole does and why it does it.